packages/kit/src/utils/fork.js

```diff
@@ -11,7 +11,9 @@ import child_process from 'node:child_process';
  * @returns {(opts: T) => Promise<U>} A function that when called starts the subprocess
  */
 export function forked(module, callback) {
-	if (process.env.SVELTEKIT_FORK) {
+	if (process.env.SVELTEKIT_FORK && process.send) {
```

Comment:
when would `process.send` be `undefined`? Could that happen for us (in which case this would also hang)?

Reply:
`process.send` is defined if it's a child process. it's kinda redundant with the `SVELTEKIT_FORK` env var (though I guess someone could do `SVELTEKIT_FORK vite build` if they wanted to deliberately break it) but we need it or TypeScript complains